library/api/src/main/java/com/epmedu/animeal/api/feeding/FeedingApiImpl.kt

```diff
@@ -43,23 +43,22 @@ internal class FeedingApiImpl(
         )
     }
 
-    override suspend fun getApprovedFeedingHistories(feedingPointId: String): ApiResult<SearchFeedingHistoriesQuery.Data> {
```

Comment:
Don't touch this code. You need to get all feeding histories for feedings screen. Create a separate method for this without any filters.

Reply:
The "status" parameter is optional, a separate method is not required

Reply:
This method filters feedings by feeding point and you don't need this at all, because the requirement is to get all feedings